src/About.vue

```diff
@@ -11,6 +11,9 @@
       <li>
         <a href="">Homework4</a>
       </li>
+      <li>
+        <a href="">Homework6</a>
+      </li>
```

Comment:
The whole `About.vue` component looks redundant. 
As far as I remember, it is not used anywhere. Please, check and remove it, if it's true.